go.mod

```diff
@@ -37,8 +37,10 @@ require (
 	google.golang.org/grpc v1.39.0
 	google.golang.org/grpc/examples v0.0.0-20210818220435-8ab16ef276a3
 	google.golang.org/protobuf v1.27.1
+	gotest.tools v2.2.0+incompatible
 	mosn.io/api v1.1.0
 	mosn.io/layotto/components v0.0.0-20220413092851-55c58dbb1a23
+	mosn.io/layotto/sdk/go-sdk v0.0.0-20220928065250-7be14b87a44e
```

Comment:
It's unreasonable to make Layotto depend on Layotto sdk. How about add a sub go module for testing tools?